README.md

```diff
@@ -1,57 +1,18 @@
-# Northcoders House of Games API
+# NC Backend Project
 
-## Background
+## Project Trello
 
```

Comment:
Make sure you tell the user they will also need to run 'npm install' to get all the needed dependencies